apps/web/pages/event-types/index.tsx

```diff
@@ -881,7 +882,7 @@ const EventTypesPage = () => {
           customLoader={<SkeletonLoader />}
           success={({ data }) => (
             <>
-              {data.eventTypeGroups.length > 1 ? (
+              {data.eventTypeGroups.length > 1 && (
```

Comment:
I would prefer showing the `EventTypeListHeading` also if we have a filtered view with only one checked item
<img width="1235" alt="Screenshot 2023-06-19 at 09 28 45" src="https://github.com/calcom/cal.com/assets/30310907/c9385f5c-6f29-43c3-9f19-00848ad29781">

Reply:
@Udit-takkar Maybe wasn't fully clear here. I meant to only show the heading if the user also has teams (but it's filtered to only one). If a user does not have teams we should show the heading as per design. If that would mean we need to fetch some extra data, we can also leave it without the heading (as you had it before)

Reply:
@CarinaWolli if a user does not have team then we do not show heading for user event types (in the design)

Reply:
But the heading is currently always shown, also without teams:
<img width="630" alt="Screenshot 2023-06-19 at 14 50 47" src="https://github.com/calcom/cal.com/assets/30310907/04db1c4c-542f-4031-91b9-f265fab833fb">

Reply:
@CarinaWolli pushed some changes fixing this